ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -72,12 +75,15 @@ public Runner(
 
   public void start() {
     final RequestMapper requestMapper = createRequestMapper(vertx);
-    httpService = new JsonRpcHttpService(responseFactory, serverOptions, requestMapper);
-    vertx.deployVerticle(httpService, this::handleDeployResult);
+    jsonRpcHttpService = new JsonRpcHttpService(responseFactory, serverOptions, requestMapper);
+    httpService = new HttpService(serverOptions);
+    vertx.deployVerticle(jsonRpcHttpService, this::jsonRpcServiceDeployment);
+    vertx.deployVerticle(httpService, this::httpServiceDeployment);
```

Comment:
Don't think the upcheck needs it's own verticle and it's own http server. There is already a non json rpc route on the jsonHttpService so it unfortunately misnamed at the moment. What benefit do we get from having this seperate?

Reply:
Which is the non Etheremy JSON-RPC route in the `jsonRpcHttpService`?

Reply:
Yeah, I may have to re-write has having only one HttpServer ...as there should be only one listening to a single port 🙄